charts/testkube-operator/values.yaml

```diff
@@ -146,6 +146,10 @@ webhook:
   enabled: true
   ## Name of the webhook
   name: webhook-admission
+  ## Limit which requests for namespaced resources are intercepted, by specifying a namespaceSelector.
+  namespaceSelector: []
+  ## Scope specifies a scope to match. Valid values are "Cluster", "Namespaced", and "*". Default is "*".
+  scope: ""
```

Comment:
check my comment for hardcoding, this won't be needed if it is hardcoded to Namespaced